### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#220] [Polls]
Currently, all translations come from Pull Requests. There are several problems with that: 
1. There are releases with translations only, which is probably not what many would expect to see.
2. There will be a lot of duplicate notes in the release notes, because during the `dev` development cycle, one language is updated several times, and the automatic release will show each "feat" and "fix" commit in the release notes.

What are benefits (or not?):
1. You can see contributors, and it is not an easy task to translate, so this way we give translators a credit by saving their work in GIt history under their names. 
2. Wasn't the first one enough?

> [!NOTE]  
> Some translation strings are functional, which means the option works only with the correct translation.

So I want to open poll here with potential fixes.

I mentioned before, that I don't want to play with crowdin, weblate, etc, if someone wants to take on this burden and do it on their own behalf, I would be glad to accept translations from those resources.

**Comment by comradekingu** [comment]
Weblate is what you want. :)
